chain/epoch_manager/src/lib.rs

```diff
@@ -142,6 +142,63 @@ impl EpochManager {
         Ok(epoch_manager)
     }
 
+    /// Only used in mock network
+    /// Copy the necessary epoch info related to `block_hash` from `source_epoch_manager` to
+    /// the current epoch manager.
+    /// Note that this function doesn't copy info stored in EpochInfoAggregator, so `block_hash` must be
+    /// the last block in an epoch in order for the epoch manager to work properly after this function
```

Comment:
Do we check that it is the last block of the epoch? what kind of error would happen if it is not?

Reply:
We don't check it in this function but we check it in setup_mock_network